### PR TITLE
DOCSUP-5266: Edit and translate to Russian

### PR DESCRIPTION
**Pull request by ka1bi4**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Documentation RU for:
- #12484
- #12662
- #12668

Detailed description / Documentation draft:

- Add RU documentation for FROM_UNIXTIME function
- Add minMap and maxMap links to SimpleAggregateFunction topic.
- Add `file()` example to clickhouse-local docs

### CODE REVIEW

**Comment by abyss7** [comment]
I'm sorry - haven't noticed, it's a draft. Fill free to discard my changes.

**Comment by ka1bi4** [comment]
> I'm sorry - haven't noticed, it's a draft. Fill free to discard my changes.

Thanks for comments, I'll approve them

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 1]
```diff
@@ -602,7 +602,7 @@ This is necessary for searching for pageviews in the corresponding session.
 
 ## formatDateTime {#formatdatetime}
 
-Function formats a Time according given Format string. N.B.: Format is a constant expression, e.g. you can not have multiple formats for single result column.
+Function formats a Time according to the given Format string. N.B.: Format is a constant expression, e.g. you cannot have multiple formats for a single result column.
```
Comment (adevyatova) [inline docs/en/sql-reference/functions/date-time-functions.md:605]:
```suggestion
Formats a Time according to the given Format string. Format is a constant expression, so you cannot have multiple formats for a single result column.
```

File: docs/ru/operations/utilities/clickhouse-local.md [hunk 2]
```diff
@@ -78,13 +79,17 @@ $ clickhouse-local --query "
 
 А теперь давайте выведем на экран объём оперативной памяти, занимаемой пользователями (Unix):
```
Comment (adevyatova) [inline docs/ru/operations/utilities/clickhouse-local.md:80]:
```suggestion
Объём оперативной памяти, занимаемой пользователями (Unix):
```

File: docs/ru/sql-reference/data-types/simpleaggregatefunction.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
 
-`SimpleAggregateFunction(name, types_of_arguments…)` data type stores current value of the aggregate function, and does not store its full state as [`AggregateFunction`](../../sql-reference/data-types/aggregatefunction.md) does. This optimization can be applied to functions for which the following property holds: the result of applying a function `f` to a row set `S1 UNION ALL S2` can be obtained by applying `f` to parts of the row set separately, and then again applying `f` to the results: `f(S1 UNION ALL S2) = f(f(S1) UNION ALL f(S2))`. This property guarantees that partial aggregation results are enough to compute the combined one, so we don’t have to store and process any extra data.
+Тип данных `SimpleAggregateFunction(name, types_of_arguments…)` хранит только текущее значение агрегатной функции и не сохраняет ее полное состояние, как это делает [`AggregateFunction`](../../sql-reference/data-types/aggregatefunction.md). Такая оптимизация может быть применена к функциям, для которых выполняется следующее свойство: результат выполнения функции `f` к набору строк `S1 UNION ALL S2` может быть получен путем выполнения `f` к отдельным частям набора строк,
```
Comment (adevyatova) [inline docs/ru/sql-reference/data-types/simpleaggregatefunction.md:3]:
```suggestion
Тип данных `SimpleAggregateFunction(name, types_of_arguments…)` хранит только текущее значение агрегатной функции и не сохраняет ее полное состояние, как это делает [`AggregateFunction`](../../sql-reference/data-types/aggregatefunction.md). Такая оптимизация может быть применена к функциям, которые обладают следующим свойством: результат выполнения функции `f` к набору строк `S1 UNION ALL S2` может быть получен путем выполнения `f` к отдельным частям набора строк,
```

File: docs/ru/sql-reference/data-types/simpleaggregatefunction.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
+а затем повторного выполнения `f` к результатам: `f(S1 UNION ALL S2) = f(f(S1) UNION ALL f(S2))`. Это свойство гарантирует, что результатов частичной агрегации достаточно для вычисления комбинированной, поэтому нам не нужно хранить и обрабатывать какие-либо дополнительные данные.
```
Comment (adevyatova) [inline docs/ru/sql-reference/data-types/simpleaggregatefunction.md:4]:
```suggestion
а затем повторного выполнения `f` к результатам: `f(S1 UNION ALL S2) = f(f(S1) UNION ALL f(S2))`. Это свойство гарантирует, что результатов частичной агрегации достаточно для вычисления комбинированной, поэтому хранить и обрабатывать какие-либо дополнительные данные не требуется.
```

File: docs/ru/sql-reference/data-types/simpleaggregatefunction.md [hunk 5]
```diff
@@ -15,15 +16,22 @@ The following aggregate functions are supported:
 -   [`groupBitXor`](../../sql-reference/aggregate-functions/reference/groupbitxor.md#groupbitxor)
 -   [`groupArrayArray`](../../sql-reference/aggregate-functions/reference/grouparray.md#agg_function-grouparray)
 -   [`groupUniqArrayArray`](../../sql-reference/aggregate-functions/reference/groupuniqarray.md#groupuniqarray)
+-   [`sumMap`](../../sql-reference/aggregate-functions/reference/summap.md#agg_functions-summap)
+-   [`minMap`](../../sql-reference/aggregate-functions/reference/minmap.md#agg_functions-minmap)
+-   [`maxMap`](../../sql-reference/aggregate-functions/reference/maxmap.md#agg_functions-maxmap)
 
-Values of the `SimpleAggregateFunction(func, Type)` look and stored the same way as `Type`, so you do not need to apply functions with `-Merge`/`-State` suffixes. `SimpleAggregateFunction` has better performance than `AggregateFunction` with same aggregation function.
+!!! note "Примечание"
+    Значения `SimpleAggregateFunction(func, Type)` отображаются и хранятся так же, как и `Type`, поэтому вам не требуется применять функции с суффиксами `-Merge`/`-State`.
```
Comment (adevyatova) [inline docs/ru/sql-reference/data-types/simpleaggregatefunction.md:24]:
```suggestion
    Значения `SimpleAggregateFunction(func, Type)` отображаются и хранятся так же, как и `Type`, поэтому комбинаторы [-Merge](../../sql-reference/aggregate-functions/combinators.md#aggregate_functions_combinators-merge) и [-State]((../../sql-reference/aggregate-functions/combinators.md#agg-functions-combinator-state) не требуются. 
```

File: docs/ru/sql-reference/data-types/simpleaggregatefunction.md [hunk 6]
```diff
@@ -15,15 +16,22 @@ The following aggregate functions are supported:
...
+    `SimpleAggregateFunction` имеет лучшую производительность, чем `AggregateFunction` с той же агрегатной функцией.
 
-**Parameters**
 
--   Name of the aggregate function.
--   Types of the aggregate function arguments.
+**Параметры**
 
-**Example**
+-   имя агрегатной функции.
```
Comment (adevyatova) [inline docs/ru/sql-reference/data-types/simpleaggregatefunction.md:31]:
```suggestion
-   `func` — имя агрегатной функции.
```

File: docs/ru/sql-reference/data-types/simpleaggregatefunction.md [hunk 7]
```diff
@@ -15,15 +16,22 @@ The following aggregate functions are supported:
...
+-   типы аргументов агрегатной функции.
```
Comment (adevyatova) [inline docs/ru/sql-reference/data-types/simpleaggregatefunction.md:32]:
```suggestion
-   `type` — типы аргументов агрегатной функции.
```

File: docs/ru/sql-reference/data-types/simpleaggregatefunction.md [hunk 8]
```diff
@@ -1,8 +1,9 @@
 # SimpleAggregateFunction {#data-type-simpleaggregatefunction}
```
Comment (adevyatova) [inline docs/ru/sql-reference/data-types/simpleaggregatefunction.md:1]:
```suggestion
# SimpleAggregateFunction(func, type) {#data-type-simpleaggregatefunction}
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 9]
```diff
@@ -654,15 +673,53 @@ formatDateTime(Time, Format\[, Timezone\])
...
 ┌─formatDateTime(toDate('2010-01-04'), '%g')─┐
 │ 10                                         │
 └────────────────────────────────────────────┘
 ```
 
+## FROM\_UNIXTIME {#fromunixtime}
+
+Функция преобразует метку времени Unix в дату. Если указан только один аргумент типа [Integer](../../sql-reference/data-types/int-uint.md), то функция действует так же, как [toDateTime](../../sql-reference/functions/type-conversion-functions.md#todatetime), и возвращает тип [DateTime](../../sql-reference/data-types/datetime.md).
+
+**Пример**
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:691]:
```suggestion
**Примеры**

Если указан только один аргумент типа [Integer](../../sql-reference/data-types/int-uint.md), то функция действует так же, как [toDateTime](../../sql-reference/functions/type-conversion-functions.md#todatetime), и возвращает тип [DateTime](../../sql-reference/data-types/datetime.md).
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 10]
```diff
@@ -654,15 +673,53 @@ formatDateTime(Time, Format\[, Timezone\])
...
+
+```text
+┌─FROM_UNIXTIME(423543535)─┐
+│      1983-06-04 10:58:55 │
+└──────────────────────────┘
+```
+
+В случае, когда есть два аргумента: первый типа [Integer](../../sql-reference/data-types/int-uint.md) или [DateTime](../../sql-reference/data-types/datetime.md), а второй является строкой постоянного формата — функция работает таким же образом, как [formatDateTime](#formatdatetime), и возвращает значение типа [String](../../sql-reference/data-types/string.md#string).
+
+**Пример**
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:709]:
```suggestion
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 11]
```diff
@@ -654,15 +673,53 @@ formatDateTime(Time, Format\[, Timezone\])
...
 
 ```
 ┌─formatDateTime(toDate('2010-01-04'), '%g')─┐
 │ 10                                         │
 └────────────────────────────────────────────┘
 ```
 
+## FROM\_UNIXTIME {#fromunixtime}
+
+Функция преобразует метку времени Unix в дату. Если указан только один аргумент типа [Integer](../../sql-reference/data-types/int-uint.md), то функция действует так же, как [toDateTime](../../sql-reference/functions/type-conversion-functions.md#todatetime), и возвращает тип [DateTime](../../sql-reference/data-types/datetime.md).
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:689]:
```suggestion
Функция преобразует метку времени Unix в дату. 
```

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 12]
```diff
@@ -661,31 +661,32 @@ Result:
 └────────────────────────────────────────────┘
 ```
 
-[Original article](https://clickhouse.tech/docs/en/query_language/functions/date_time_functions/) <!--hide-->
-
 ## FROM\_UNIXTIME {#fromunixfime}
 
-When there is only single argument of integer type, it act in the same way as `toDateTime` and return [DateTime](../../sql-reference/data-types/datetime.md).
-type.
+Function converts Unix timestamp to date. When there is only a single argument of [Integer](../../sql-reference/data-types/int-uint.md) type, it acts in the same way as [toDateTime](../../sql-reference/functions/type-conversion-functions.md#todatetime) and return [DateTime](../../sql-reference/data-types/datetime.md) type.
```
Comment (adevyatova) [inline docs/en/sql-reference/functions/date-time-functions.md:666]:
```suggestion
Function converts Unix timestamp to a calendar date and a time of a day. When there is only a single argument of [Integer](../../sql-reference/data-types/int-uint.md) type, it acts in the same way as [toDateTime](../../sql-reference/functions/type-conversion-functions.md#todatetime) and return [DateTime](../../sql-reference/data-types/datetime.md) type.
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 13]
```diff
@@ -654,15 +673,54 @@ formatDateTime(Time, Format\[, Timezone\])
...
 
 ```
 ┌─formatDateTime(toDate('2010-01-04'), '%g')─┐
 │ 10                                         │
 └────────────────────────────────────────────┘
 ```
 
+## FROM\_UNIXTIME {#fromunixtime}
+
+Функция преобразует метку времени Unix в дату.
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:689]:
```suggestion
Функция преобразует Unix timestamp в календарную дату и время. 
```

File: docs/en/sql-reference/data-types/simpleaggregatefunction.md [hunk 14]
```diff
@@ -31,11 +35,7 @@ Values of the `SimpleAggregateFunction(func, Type)` look and stored the same way
 **Example**
 
 ``` sql
-CREATE TABLE t
-(
-    column1 SimpleAggregateFunction(sum, UInt64),
-    column2 SimpleAggregateFunction(any, String)
-) ENGINE = ...
+CREATE TABLE simple (id UInt64,val SimpleAggregateFunction(sum,Double)) ENGINE=AggregatingMergeTree ORDER BY id;
```
Comment (adevyatova) [inline docs/en/sql-reference/data-types/simpleaggregatefunction.md:38]:
```suggestion
CREATE TABLE simple (id UInt64, val SimpleAggregateFunction(sum, Double)) ENGINE=AggregatingMergeTree ORDER BY id;
```

File: docs/ru/operations/utilities/clickhouse-local.md [hunk 15]
```diff
@@ -76,7 +77,9 @@ $ clickhouse-local --query "
 1	2
 ```
 
-А теперь давайте выведем на экран объём оперативной памяти, занимаемой пользователями (Unix):
+Объём оперативной памяти, занимаемой пользователями (Unix):
```
Comment (adevyatova) [inline docs/ru/operations/utilities/clickhouse-local.md:80]:
```suggestion
Объём оперативной памяти, занимаемой процессами, которые запустил пользователь (Unix):
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 16]
```diff
@@ -654,15 +673,54 @@ formatDateTime(Time, Format\[, Timezone\])
...
+
+Результат:
+
+```text
+┌─FROM_UNIXTIME(423543535)─┐
+│      1983-06-04 10:58:55 │
+└──────────────────────────┘
+```
+
+В случае, когда есть два аргумента: первый типа [Integer](../../sql-reference/data-types/int-uint.md) или [DateTime](../../sql-reference/data-types/datetime.md), а второй является строкой постоянного формата — функция работает таким же образом, как [formatDateTime](#formatdatetime), и возвращает значение типа [String](../../sql-reference/data-types/string.md#string).
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:709]:
```suggestion
В случае, когда есть два аргумента: первый типа [Integer](../../sql-reference/data-types/int-uint.md) или [DateTime](../../sql-reference/data-types/datetime.md), а второй является строкой постоянного формата — функция работает также, как [formatDateTime](#formatdatetime), и возвращает значение типа [String](../../sql-reference/data-types/string.md#string).
```

File: docs/ru/sql-reference/data-types/simpleaggregatefunction.md [hunk 17]
```diff
@@ -15,22 +16,24 @@ The following aggregate functions are supported:
...
-CREATE TABLE t
-(
-    column1 SimpleAggregateFunction(sum, UInt64),
-    column2 SimpleAggregateFunction(any, String)
-) ENGINE = ...
+CREATE TABLE simple (id UInt64, val SimpleAggregateFunction(sum, Double)) ENGINE=AggregatingMergeTree ORDER BY id;
 ```
 
-[Original article](https://clickhouse.tech/docs/en/data_types/simpleaggregatefunction/) <!--hide-->
+[Оригинальная статья](https://clickhouse.tech/docs/en/data_types/simpleaggregatefunction/) <!--hide-->
```
Comment (vdimir) [inline docs/ru/sql-reference/data-types/simpleaggregatefunction.md:39]:
```suggestion
[Оригинальная статья](https://clickhouse.tech/docs/en/sql-reference/data-types/simpleaggregatefunction/) <!--hide-->
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 18]
```diff
@@ -305,14 +305,19 @@ WITH toDateTime64('2020-01-01 10:20:30.999', 3) AS dt64 SELECT toStartOfSecond(d
 
 Переводит дату-с-временем или дату в число типа UInt16, содержащее номер ISO года. ISO год отличается от обычного года, потому что в соответствии с [ISO 8601:1988](https://en.wikipedia.org/wiki/ISO_8601) ISO год начинается необязательно первого января.
 
-Пример:
+**Пример:**
```
Comment (vdimir) [inline docs/ru/sql-reference/functions/date-time-functions.md:308]:
```suggestion
**Пример**
```

**Review by vdimir — APPROVED** [review]

**Comment by vdimir** [comment]
```
INFO:root:Tag <a href="#type_conversion_function-reinterpretAsString">Interprets</a>
INFO:root:Link to nowhere: #type_conversion_function-reinterpretAsString
INFO:root:Tag <a href="#type_conversion_function-reinterpretAsString">interprets</a>
INFO:root:Link to nowhere: #type_conversion_function-reinterpretAsString
WARNING:root:Found 259 duplicate anchor points
ERROR:root:Found 2 links to nowhere in en
```

Seems that error not connected with changes, need to merge `master`. I cannot push it because `Allow edits and access to secrets by maintainers` is not set in this PR (in the bottom of right column in github web ui on page with this PR).

**Comment by ka1bi4** [comment]
Allow edits and access to secrets by maintainers — was enabled for all time.

**Pull request merged**